ooiui/static/js/partials/StreamDownloadForm.html

```diff
@@ -10,10 +10,13 @@
         <img width="50px" src="/img/logos-banners/OOI_Logo.svg">
 
 
-          <span style="margin-left:25%">
+          <span class="subscription-selection" style="margin-left:25%">
               <button style="padding-top:5px;" id="subscription-selection-select" type="button" class="btn btn-default btn-xs">
                 <span id="subscription-selection-icon" class="fa fa-heart-o" aria-hidden="true"></span> Enable Subscription
               </button>
+              <span class="subscribe-info">
+                  <i class="fa fa-info-circle" data-title="Enabling subscriptions will keep you updated when ever a stream changes."></i>
```

Comment:
Should read "Enabling subscription to a stream will cause the data provision service to auto-generate this file once a day.  The user may retrieve it on one of the OPeNDAP servers." or equivalent.